### PR TITLE
Header Updated

### PR DESCRIPTION
**Pull request by Anrsgrl**
- HeaderMenu fixed
- Scroll problem fixed (when we open HeaderMenu)
- Header icons fixed
- Cart counter added
- Search modal added 

> ( -responsive -variables and mixins used )

- Search modal animations added
- Cart aside added 

> ( not finished, it is in "display:none" mode i will change it a bit later )

### CODE REVIEW

**Comment by Anrsgrl** [comment]
[Search Modal at header](https://trello.com/c/jdyxpoAU/32-search-modal-at-header)

File: src/App.tsx [hunk 1]
```diff
@@ -33,6 +34,7 @@ function App() {
           <Route path="/orderTracking" element={<OrderTracking />} />
           <Route path="/wishlist" element={<Whishlist />} />
           <Route path="/checkout" element={<CheckOut />} />
+          <Route path="/myaccount" element={<MyAccount />} />
```
Comment (Negodyaj) [inline src/App.tsx:37]: please change to /my-account

File: src/components/CartAside/CartAside.scss [hunk 2]
```diff
@@ -0,0 +1,138 @@
+@import '../../vendors/mixins';
+@import '../../vendors/variables';
+
+.CartAside-field{
```
[the remark this answers is not in the document]
Comment (Anrsgrl) [reply src/components/CartAside/CartAside.scss]: Done

File: src/components/SearchModal/SearchModal.scss [hunk 3]
```diff
@@ -0,0 +1,150 @@
+@import '../../vendors/mixins';
+@import '../../vendors/variables';
+
+
+.search-open{
+    display: block !important;
+    opacity: 1 !important;
+    visibility: visible !important;
+}
+.SearchModal-field{
```
[the remark this answers is not in the document]
Comment (Anrsgrl) [reply src/components/SearchModal/SearchModal.scss]: Done

File: src/App.tsx (hunk 1 above)
Comment (Anrsgrl) [reply src/App.tsx]: Done

File: src/components/CartAside/CartAside.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                    <div className="list-img">
+                        <img src={bowl} alt="bowl" />
+                    </div>
+                    <div className="list-content">
+                        <p>Keny Bowl</p>
+                        <p>QTY: 2</p>
+                        <p>$25.00</p>
+                    </div>
+                </li>
+            </ul>
```
[the remark this answers is not in the document]
Comment (Anrsgrl) [reply src/components/CartAside/CartAside.tsx:32]: Don't check it yet, it is cart's aside, elements gonna be listing in ul when we click cart button on carts

**Pull request merged**